sql/R__transfer_credits.sql

```diff
@@ -199,7 +199,7 @@ begin
 
   -- retire credits if auto_retire true
   if auto_retire = true then
-    perform retire_credits(vintage_id, buyer_wallet_id, address_id, units);
+    perform retire_credits(vintage_id, buyer_wallet_id, address_id, units, default);
```

Comment:
this makes the tests fail, if you apply the change from above, you should be able to just use:
```suggestion
    perform retire_credits(vintage_id, buyer_wallet_id, address_id, units);
```